banderwagon/element.go

```diff
@@ -42,6 +42,41 @@ func (p Element) Bytes() [sizePointCompressed]byte {
 	return x.Bytes()
 }
 
+func BatchNormalize(elements []*Element) {
```

Comment:
Just for good measure, could you add a test for the point at infinity and duplicated elements

Reply:
Done!